models/quality_measures/intermediate/cbe0055_diabetes_eye_exam/quality_measures__int_cbe0055_long.sql

```diff
@@ -76,9 +76,14 @@ with denominator as (
                 , denominator.performance_period_end
                 , denominator.measure_id
                 , denominator.measure_name
-            order by
-                  numerator.evidence_date desc nulls last
-                , exclusions.exclusion_date desc nulls last
+--            order by
+--                  numerator.evidence_date desc nulls last
+--                , exclusions.exclusion_date desc nulls last
+              order by
+                  case when numerator.evidence_date is null then 1 else 0 end,
+                  numerator.evidence_date desc
+                , case when exclusions.exclusion_date is null then 1 else 0 end,
+                  exclusions.exclusion_date desc
```

Comment:
Should we remove the commented-out section? Any reason to keep it?

Reply:
Do we need to keep the commented-out section? Just worried this will cause confusion.

Reply:
Removed.

Reply:
Sorry, I only commented on one model. All of the "long" models for quality_measures have the commented-out section in the order by clause.

Reply:
Removed from all long models.